default.json

```
@@ -104,6 +104,16 @@
       "datasourceTemplate": "docker",
       "versioningTemplate": "docker"
     },
+    {
+      "description": "Update Github Action references in the markdown files",
+      "fileMatch": ["\\.md$"],
+      "matchStrings": [
+        "\\suses: (?<depName>actions\\/checkout)@(?<currentValue>v\\d+\\.\\d+\\.\\d+)",
+        "\\suses: (?<depName>renovatebot\\/github-action)@(?<currentValue>v\\d+\\.\\d+\\.\\d+)"
```

```suggestion
        "\\suses: (?<depName>[\\w-]+\\/[\\w-]+)(?<path>\\/.*)?@(?<currentValue>v\\d+\\.\\d+\\.\\d+)"
```

Allow updating any action, so we don't need to add every action we use.
Maybe we should add another regex for pinned actions?

---

> Maybe we should add another regex for pinned actions?

That would be nice in case we ever do want to use pinned actions in our Markdown files.
Not sure if that would cause conflicts with the current regex?